torch/quantization/fx/quantization_patterns.py

```diff
@@ -597,9 +600,10 @@ def convert(self,
                     if isinstance(self.conv, torch.nn.intrinsic._FusedModule):
                         float_conv = self.conv[0]
                     assert qconfig is not None
-                    float_conv.weight_post_process = qconfig.weight()
+                    weight_post_process = qconfig.weight()
                     # run weight observer
-                    float_conv.weight_post_process(float_conv.weight)
+                    weight_post_process(float_conv.weight)
+                float_conv.weight_qparams = get_qparam_dict(weight_post_process)
```

Comment:
is there an example how this will be used?  I see it set, but not how it is queried.

I think the additional complexity will depend on the callsite and how it does serialization.

Reply:
It will be used in the backend lowering pass to generate quantize node for weight, I don't have example code currently. basically the weight_qparams dictionary will be: {'qschem': ..., 'scale': ..., 'zero_point': ..., 'dtype': ...., ....} and backend lowering pass can use this info to quantize the weight.

I don't think this is additional complexity, why do you think this is additional complexity?

Reply:
is there going to be a need for custom versioned setstate/getstate functions to hold this information?

Reply:
oh I see, yeah probably, I haven't verified that. I think it's still worth it though since otherwise the backend lowering pass would need to extrac t this info, this also exposes less implementation detail to the backend lowering pass as well